apps/sql-server/fdw/fdw/__init__.py

```diff
@@ -0,0 +1,289 @@
+from multicorn import TableDefinition, ColumnDefinition, ForeignDataWrapper
+import sys
+from datetime import datetime
+from aperturedb.CommonLibrary import create_connector
+import logging
+import os
+import json
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, force=True,)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+handler = logging.FileHandler("/tmp/fdw.log")
+handler.setFormatter(logging.Formatter(
+    "%(asctime)s %(levelname)s %(message)s"))
+logger.addHandler(handler)
+logger.propagate = False
+
+
+def load_aperturedb_env(path="/app/aperturedb.env"):
+    """Load environment variables from a file.
+    This is used because FDW is executed in a "secure" environment where
+    environment variables cannot be set directly.
+    """
+    if not os.path.exists(path):
+        raise RuntimeError(f"Missing environment file: {path}")
+    with open(path) as f:
+        for line in f:
+            if not line.strip() or line.startswith("#"):
+                continue
```

Comment:
do we maybe want to use something like python-dotenv for passing the environment?